.github/workflows/ci.yml

```diff
@@ -1,30 +1,135 @@
----
-
 name: CI
 
 on:
   pull_request:
-    branches: [devel]
+    branches:
+      - devel
```

Comment:
I think if you just leave off the `branches` part, it will trigger for a pull request on _any_ branch, which would be what we want for backports to be tested properly.

Reply:
This evaluates the base branch, not the source branch name. Limiting this workflow to only devel and release branches would prevent it  from running against other branches that get pushed to the repo. There shouldn't be many (any?) of those, though.

From the [GHA docs](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#onpushpull_requestbranchestags):
> For a pull_request event, only branches and tags on the base are evaluated.

Reply:
Yep, Sam is correct: the `pull_request` event happens whenever there's an interaction with a PR. If there's a branch pushed to the upstream repo, it'll generate a `push` event, and only after a PR gets created there's *additionally/separately* a `pull_request` happening. If somebody would want to send a PR against a non-matching branch, this would not cause GHA to react to this specific event if the branch is not included.
Although, it's hard to imagine this happening often (or ever) so I don't see a strong argument for maintaining the list of branches for this event.

Reply:
I'll remove the branch list.